warehouse/ingest-core/src/main/java/datawave/ingest/data/tokenize/StandardAnalyzer.java

```diff
@@ -1,29 +1,29 @@
 package datawave.ingest.data.tokenize;
 
-import java.io.IOException;
 import java.io.Reader;
 
 import org.apache.lucene.analysis.TokenStream;
 import org.apache.lucene.analysis.core.LowerCaseFilter;
 import org.apache.lucene.analysis.core.StopAnalyzer;
 import org.apache.lucene.analysis.core.StopFilter;
-import org.apache.lucene.analysis.standard.StandardFilter;
-import org.apache.lucene.analysis.util.CharArraySet;
-import org.apache.lucene.analysis.util.StopwordAnalyzerBase;
-import org.apache.lucene.util.Version;
+import org.apache.lucene.analysis.en.EnglishAnalyzer;
+import org.apache.lucene.analysis.standard.ClassicFilter;
+import org.apache.lucene.analysis.CharArraySet;
+import org.apache.lucene.analysis.StopwordAnalyzerBase;
 
 /**
- * Filters {@link StandardTokenizer} with {@link StandardFilter}, {@link LowerCaseFilter} and {@link StopFilter}, using a list of English stop words (unless
+ * Filters {@link StandardTokenizer} with {@link LowerCaseFilter} {@link ClassicFilter} and {@link StopFilter}, using a list of English stop words (unless
  * otherwise specified).
  * <p>
  * This analyzer does NOT provide the ability to apply BASIS RLP processing
- * 
  */
 public class StandardAnalyzer extends StopwordAnalyzerBase {
     
     /** Default maximum allowed token length */
     public static final int DEFAULT_MAX_TOKEN_LENGTH = 255;
-    public static final int DEFAULT_TRUNCATE_TOKEN_LENGTH = 1024;
+    
+    /** Default token truncation length */
+    public static final int DEFAULT_TRUNCATE_TOKEN_LENGTH = 64;
```

Comment:
The original constant wasn't actually used anywhere and a duplicate constant was set in `StandardTokenizer` to a value of 50 and used throughout the code. This has been updated to 64 based on discussions to accommodate slightly longer strings.